src/msg91OTP.js

```diff
@@ -55,7 +55,8 @@ class Msg91Otp {
         otp_expiry: args.expiry || this.defaultOtpExpiry,
         otp_length: args.otpLength || this.defaultOtpLength,
         email: args.email || '',
-        invisible: args.invisible || 1
+        invisible: args.invisible || 1,
+        extra_param: args.extra_param || ''
```

Comment:
@sreeteja06 is this backward compatible with API version less than v5?

Reply:
@sumitsk20  yes works with v1, v2, v3, v4, and v5.